spec/syntax.md

```diff
@@ -714,7 +714,7 @@ markup = "{" [s] "#" identifier *(s option) *(s attribute) [s] ["/"] "}"  ; open
 > {#button}Submit{/button} or {#img alt=|Cancel| /}.
 > ```
 
-> A _message_ with attributes in the closing tag:
+> A _message_ with options on the closing tags:
```

Comment:
The opening tag has options too. Perhaps a bit more prose to show why we have this example?

```suggestion
> A _message_ containing _markup_ that uses _options_ to pair 
> two closing markup _placeholders_ to the one open markup _placeholder_:
```

Question: should this example include `u:id`?